components/calculation/PopulationComparisonTable.tsx

```diff
@@ -145,6 +148,15 @@ export default function PopulationComparisonTable({ patientId }: PopulationCompa
             })}
           </tbody>
         </Table>
+        <Autocomplete
+          data={Object.keys(defIds)}
```

Comment:
Is it possible to make the defIds that pop up scrollable? When I first click on the input the following are displayed but in order to see the others, I have to start typing:

![image](https://github.com/projecttacoma/fqm-testify/assets/30158156/974c0a31-381f-4322-b0db-a4e8847a7ce0)